templates/poster.html

```diff
@@ -123,6 +123,33 @@ <h5 style="color: red;">
     })
 </script>
 
+<div class="container" style="padding-bottom: 30px; padding-top:30px">
+  <center>
+    <h2> Similar Papers </h2>
+  </center>
+</div>
+<p></p>
+<div  class="container" >
+  <div class="row">
+  {% for openreview in paper_recs %}
+    <div class="col-md-4 col-xs-6">
+      <div class="pp-card" >
+        <div class="pp-card-header" class="text-muted">
+          <a href="poster_{{openreview.content.iclr_id}}.html" class="text-muted">
+            <h5 class="card-title" align="center">{{openreview.content.title}}</h5>
+          </a>
+          <h6 class="card-subtitle text-muted" align="center">
+             {% for a in openreview.content.authors %}
+             {{a}},
+             {% endfor %}
+          </h6>
+          <center><img class="cards_img" src="https://iclr.github.io/iclr-images/{{openreview.content.iclr_id}}.png" width="80%"/></center>
```

Comment:
Can we make `https://iclr.github.io/iclr-images` a config variable and use `{{ config.xxxx_url }}` here?
Then we can edit the `sitedata_acl2020/config.yml` file to add this field.

Reply:
Updated with latest commit